packages/powersync/lib/src/database/powersync_db_mixin.dart

```diff
@@ -78,7 +75,6 @@ mixin PowerSyncDatabaseMixin implements SqliteConnection {
         .cast<UpdateNotification>();
 
     await database.initialize();
-    await migrations.migrate(database);
     await updateSchema(schema);
```

Comment:
The `hasSynced` init should be added here https://github.com/powersync-ja/powersync.dart/blob/9e1e376a1b1c18c8347a3c2a3bd056731296f305/packages/powersync/lib/src/powersync_database.dart#L158